packages/sdk/src/auth/token-provider.ts

```diff
@@ -74,13 +79,19 @@
     if (existingDelegatePromise) {
       return existingDelegatePromise;
     }
-    const delegatePromise = this.delegate.get().then((it) => {
+    
+    const delegatePromise = this.delegate.get().then(async (it) => {
       this.tokenStore.save(this.subject, it.rawValue);
+      const walletDto: WalletDto = {
+        id: '',
```

Comment:
I think this dto is not needed at all, since it's not defined in data-service controller